backend/utils/sidebar.py

```diff
@@ -117,10 +117,11 @@ async def __get_joined_rooms(
         starred_rooms = []
         user_rooms = user_rooms or []
         for room in user_rooms:
-            room_profile = await self.__get_room_profile(
-                member_id, room, org_members, DB
-            )
-            rooms.append(room_profile)
+            if room.get("room_members").get(member_id).get("closed") is False:
```

Comment:
We could also write it this way:
```suggestion
            if not room.get("room_members").get(member_id).get("closed"):
```

Reply:
is there a reason why using "not" is better or is it just a matter of preference?

Reply:
The pythonic way is to not explicitly compare to True /False if we don't need to, as it's the case here.

Also, negating an expr seems faster than comparing it to False. 

Kindly take a look at this article :

https://switowski.com/blog/checking-for-true-or-false

@mykie88

Reply:
thanks @zxenonx

Reply:
Yeah @mykie88, I can see you read it fully.
And obviously, you're not wrong at all. 
I just find the "if not" expr more straightforward.
But what you've written is absolutely right, so there's no problem at all.

Reply:
Thanks boss 🙏